test/unit/browser/cache/cache_test.dart

```diff
@@ -563,14 +594,72 @@ void main() {
   });
 }
 
-class MockCachingStrategy extends Mock
-    implements CachingStrategy<String, Object> {
-  MockCachingStrategy() {
-    when(this.onDidGet(typed(any), typed(any)))
-        .thenReturn(new Future.value(null));
-    when(this.onDidRelease(typed(any), typed(any), typed(any)))
-        .thenReturn(new Future.value(null));
```

Comment:
If you use `thenAnswer(() => new Future.value(null));` instead of `thenReturn...` this should work.